assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -193,22 +199,29 @@ export default function SetupMainPAX( { finishSetup } ) {
 		setShowPaxAppQueryParam( PAX_SETUP_STEP.LAUNCH );
 	}, [ navigateTo, setShowPaxAppQueryParam, hasAdwordsScope, oAuthURL ] );
 
-	const onLaunch = useCallback( ( app ) => {
-		paxAppRef.current = app;
-	}, [] );
+	const onLaunch = useCallback(
+		async ( app ) => {
+			await trackEvent( `${ viewContext }_pax`, 'pax_launch' );
+			paxAppRef.current = app;
+		},
+		[ viewContext ]
+	);
 
-	const onSetupCallback = useCallback( () => {
+	const onSetupCallback = useCallback( async () => {
 		if ( isWooCommerceActivated && ! isWooCommerceRedirectModalDismissed ) {
 			setOpenDialog( true );
 			return;
 		}
 
+		await trackEvent( `${ viewContext }`, 'start_setup_pax' );
```

Comment:
```suggestion
		// awaiting because `createAccount` may trigger a navigation.
		await trackEvent( viewContext, 'start_setup_pax' );
```
Also, no need to interpolate `viewContext` with nothing additional.